src/Sulu/Bundle/MediaBundle/Resources/public/js/services/overlay-manager.js

```diff
@@ -12,6 +12,7 @@ define(function() {
     'use strict';
 
     var instance = null,
+        overlayOpened = false,
```

Comment:
@marcelmoosbrugger unfortunately that is not possible because the start-overlay-functions are executed in the view-component context